src/components/event-details/EventDetails.tsx

```diff
@@ -106,9 +108,21 @@ const EventListing: React.FC<EventProps> = ({ thumbnail, title, details }) => {
           <GreenButton>Register</GreenButton>
         </Col>
         <Col span={10}>
-          <Title level={5}>Announcements</Title>
+          <Title level={5}>Updates for this Event</Title>
           <AnnouncementBox>
-            There are no announcements for this event
+            {announcements.length > 0
+              ? announcements.map((announcement, i) => {
+                  return (
+                    <AnnouncementCard
+                      key={i}
+                      imageSrc={announcement.imageSrc || DEFAULT_IMAGE}
```

Comment:
Do we need to use DEFAULT_IMAGE here? I think we check if imageSrc is undefined inside the AnnoucementsCard component already so this might be redundant

Reply:
Currently it is not applying a default image in AnnouncementCard, because I just recently figured out that we're doing the default image thing. I will do it in there and remove it in all the places where calling AnnouncementCard